docs/machine-learning/ml-feasibility-study.md

```diff
@@ -37,8 +37,9 @@ Collaboration from individuals with diverse skill sets is desired at this stage,
 
 ### Data discovery
 
-* Hold a data exploration workshop and deep dive with domain experts
+* Hold a [data exploration](./ml-data-exloration.md) workshop and deep dive with domain experts
 * Understand the data dictionary, data quality and access (availability)
+* Understand the quality of the data. Is there already a data validation strategy in place?
```

Comment:
Data quality is already mentioned in the previous bullet. Consider removing it from there.

Reply:
Good catch! I split up data discovery bullet point on availability, dictionary and quality.